include/beast/http/basic_parser_v1.hpp

```diff
@@ -412,6 +431,24 @@ class basic_parser_v1
     needs_eof(std::false_type) const;
 
     template<class C>
+    class has_on_start_t
+    {
+        template<class T, class R =
+            decltype(std::declval<T>().on_start(
+                std::declval<error_code&>()),
+                    std::true_type{})>
+        static R check(int);
```

Comment:
Linux only. Yeah, getting traits to work on windows must have been painful. Might be better to leave well enough alone.